db/migrate/20191118130830_create_spaces.rb

```diff
@@ -0,0 +1,16 @@
+class CreateSpaces < ActiveRecord::Migration[5.2]
+  def change
+    create_table :spaces do |t|
+      t.references :user, foreign_key: true
```

Comment:
we could rename it to 'owner' at a certain point

Reply:
ok thank you well noted
class ChangeColumnNameUsers < ActiveRecord::Migration[5.2]
 def change
   rename_column :users, :user_id, :receiver_id
 end
end